src/content/docs/infrastructure/prometheus-integrations/install-configure-prometheus-configurator/scale-prometheus-instances-instances.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: Scaling Prometheus Agent instances
+tags:
+  - Integrations
+  - Prometheus integrations
+  - Install and configure Prometheus Configurator
+metaDescription: TODO
+---
+While our clusters keep growing in size, there will be more data gathered by Prometheus and at some point we can reach the maximum memory of the node; most of the time related to the cardinality of the time series.
```

Comment:
```suggestion
When clusters grow in size, there will be more data gathered by Prometheus and at some point it will reach the limits of how much data the Prometheus agent can process. The most common failure mode is running out of memory due to the increased cardinality of the time series.
```

Reply:
Thanks!